pom.xml

```diff
@@ -228,25 +227,6 @@
                     <compatibleSinceVersion>3.0</compatibleSinceVersion>
                 </configuration>
             </plugin>
-            <plugin>
-                <artifactId>maven-enforcer-plugin</artifactId>
-                <executions>
-                    <execution>
-                        <id>display-info</id>
-                        <configuration>
-                            <rules>
-                                <enforceBytecodeVersion>
-                                    <excludes combine.children="append">
```

Comment:
No longer necessary after https://github.com/jenkinsci/plugin-pom/pull/136 (confirmed that on parent pom 3.28 the config is necessary, but on 3.29 it is not).